mmdet3d/structures/bbox_3d/cam_box3d.py

```diff
@@ -291,14 +291,19 @@ def convert_to(self, dst, rt_mat=None):
                 The conversion from ``src`` coordinates to ``dst`` coordinates
                 usually comes along the change of sensors, e.g., from camera
                 to LiDAR. This requires a transformation matrix.
-
+            correct_yaw (bool): If convert the yaw angle to the target
```

Comment:
If -> Whether

Reply:
I think the docstring `If convert the yaw angle to the target coordinate` is not proper, since this is always a `yes` answer and would cause confusion.